Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+ENV LC_ALL=C.UTF-8 \
+    LANG=C.UTF-8
+
+RUN apt-get update && \
```

Comment:
If we're updating every time we build the image, is it possible we will build an image with a new, updated version of a package that is incompatible with our app?

Reply:
That's a great point. According to [this](https://askubuntu.com/questions/222348/what-does-sudo-apt-get-update-do#222352) stackoverflow post, `apt-get update` doesn't install any new packages, it just makes `apt` aware of the latest versions of packages. Your point still stands though. Since we're installing a generic `python3-pip`, it's likely the latest version of python. Let me see if we can specify a specific version like `python3.7`.

Reply:
Done! Great catch! This could have been a source of potentially _nasty_ bugs down the line. Glad you caught it early! Re-requested your review.